lib/src/value_accessors/formatted_datetime_value_accessor.dart

```diff
@@ -0,0 +1,25 @@
+// Copyright 2020 Joan Pablo Jiménez Milian. All rights reserved.
+// Use of this source code is governed by the MIT license that can be
+// found in the LICENSE file.
+
+import 'package:intl/intl.dart';
+import 'package:reactive_forms/reactive_forms.dart';
+
+class FormattedDateTimeValueAccessor
+    extends ControlValueAccessor<DateTime, String> {
+  final DateFormat dateTimeFormat;
```

Comment:
I prefer the name of *format* instead of *dateTimeFormat*

Reply:
Then we will have `format.format` at line 16)

Reply:
You are right. So the best name is the one you have.